molecule/config.py

```diff
@@ -185,6 +293,7 @@ def env(self):
         return {
             'MOLECULE_DEBUG': str(self.debug),
             'MOLECULE_FILE': self.molecule_file,
+            'MOLECULE_BASE_FILE': self.molecule_base_file,
```

Comment:
Needs added to tests.

https://github.com/metacloud/molecule/blob/master/test/unit/test_config.py#L286